aat/main_test.go

```diff
@@ -51,12 +46,41 @@ func TestMain(m *testing.M) {
 	//router.DebugEnabled = true
 	router.SetLogger(rtrLogger)
 
+	crAuth, err := auth.NewCRAuthenticator(&testCRAuthenticator{})
+	if err != nil {
+		panic(err)
+	}
+
 	// Create router instance.
-	nxr = router.NewRouter(autoRealm, strictURI)
-	nxr.AddRealm(wamp.URI(testRealm), anonAuth, allowDisclose)
+	routerConfig := &router.RouterConfig{
+		RealmConfigs: []*router.RealmConfig{
+			&router.RealmConfig{
+				URI:           wamp.URI(testRealm),
+				StrictURI:     false,
+				AnonymousAuth: true,
```

Comment:
How do most apps you know of use WAMP authentication?  Is anonymous the usual, or the exception?  If anonymous in normally the case, then maybe best to default it to allowed.